docs/troubleshoot/upgrade-zowe-no-downtime.md

```diff
@@ -0,0 +1,91 @@
+# Upgrade Zowe in High availability
+
+As a systems programmer, you will need to upgrade Zowe regularly, to make sure you are using the latest features and to make sure you are using the latest security patches.
+
+This article describes the requirements and procedure to perform an upgrade of a Zowe installation in high availability mode, avoiding downtime.
+
+## Assumptions
+
+The following are assumptions about the installation prior to starting the upgrade process. Other settings are supported with adapted instructions.
+
+- Zowe is installed in a single shared zFS.
+- Zowe is setup in high availability mode with a sysplex distributor and two instances (A and B)
+- The upgraded instance will replace one of the existing instances (for the purposes of this procedure this will be instance B)
+- Instance A and B runs Zowe 2.4.0 and the upgraded instance will run a newer minor version (i.e. 2.10.0)
+
+## Restrictions
+
+Upgrading an installation in high availability means that for a period of time, one or more instances will run at different maintainance levels. This possibility comes with a few restrictions:
+
+- Only one API Catalog can be active at any moment
+- HA Settings need to be the same across both installations
+- Each intance has its own configuration file (e.g. `zowe.yaml` file) and the HA and security settings need to be kept in sync.
+
+## Procedure
+
+The following procedure to perform the upgrade in high availability mode is divided in two stages, first installing a new Zowe instance with the upgraded version and then the procedure to switch traffic without incurring downtime.
+
+### New installation
+
+1. Perform a separate installation of the new version of Zowe to be upgraded to, using convenience or SMP/E builds. Follow the instructions in <!-- TODO link to existing installation process --> to perform this installation
```

Comment:
... to, using any of the supported installation method ...

And would ink to https://docs.zowe.org/stable/user-guide/install-zos instead of enumerating the methods.